insilico_panels/KK/MAD_och_MAD-liknande_defekter.v1.0/README.md

```diff
@@ -0,0 +1,12 @@
+## MAD och MAD-liknande defekter
+This panel was created 2022-02-03 and contains part of the genes in “mixed small panels”. The gene list in “Mixed small panels” was divided into 5 panels. Two of the genes were added to the already existing panel “Neurometabola sjukdomar “ and the rest were put into the following new panels:
+
+Rubbningar i Kopparmetabolism
```

Comment:
I'd suggest complete names here like "MAD_och_MAD-liknande_defekter.v1.0"

Reply:
Do you mean for the title of the README as well as panel names within the README text or do you mean the name of the README file?

Reply:
I mean the name of the panels you are referring to in the README.
The title can stay as README